src/status_im/chat/models/message.cljs

```diff
@@ -137,10 +137,11 @@
                [{:ms 100 :dispatch [:chat/add-senders-to-chat-users (vals senders)]}]))}))
 
 (defn reduce-js-statuses [db ^js message-js]
-  (let [chat-id (.-localChatId message-js)
+  (let [chat-id             (.-localChatId message-js)
         profile-initialized (get-in db [:pagination-info chat-id :messages-initialized?])
-        timeline-message (timeline-message? db chat-id)]
-    (if (or profile-initialized timeline-message)
+        timeline-message    (timeline-message? db chat-id)
+        old-message         (get-in db [:messages chat-id (.-id message-js)])]
+    (if (and (or profile-initialized timeline-message) (nil? old-message))
```

Comment:
could you elaborate ? im not sure how new messages will be added then ?

Reply:
Hi, this flag only checks if we already have that message and skips that, otherwise just normal flow.

Reply:
oh right, just misread it, thank you